src/fitnesse/testsystems/slim/SlimClientBuilder.java

```diff
@@ -179,12 +182,12 @@ protected String determineSlimHost() {
     return slimHost == null ? "localhost" : slimHost;
   }
 
-  protected String getSlimFlags() {
+  protected String[] getSlimFlags() {
     String slimFlags = getVariable("slim.flags");
     if (slimFlags == null) {
       slimFlags = getVariable(SLIM_FLAGS);
     }
-    return slimFlags == null ? "" : slimFlags;
+    return slimFlags == null ? new String[] {} : slimFlags.trim().split("\\s");
```

Comment:
There is a smarter parser in ClientBuilder.parseCommandLine(). That will take into account quoted values.